### PR TITLE
7zr: fix Japanese translation

### PR DESCRIPTION
**Pull request by Ry0taK**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Japanese translation of 7zr were bit broken (e.g. `道/に/ファイルまたはディレクトリ` means `File or directory on the road`), so I tweaked it a bit.

### CODE REVIEW

File: pages.ja/common/7zr.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # 7zr
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> .7zファイルをアーカイブタイプをサポートする
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
+> `7z`と似ていますが、`7zr`は`.7z`形式のみをサポートしています。
```
Comment (blueskyson) [inline pages.ja/common/7zr.md:4]:
```suggestion
> `7z` と似ていますが、`7zr` は `.7z` 形式のみをサポートしています。
```

Comment (Ry0taK) [reply pages.ja/common/7zr.md]: Thank you for the review! I've applied the suggestion.

**Review by blueskyson — APPROVED** [review]

File: pages.ja/common/7zr.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # 7zr
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> .7zファイルをアーカイブタイプをサポートする
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
```
Comment (navarroaxel) [inline pages.ja/common/7zr.md:3]: I don't speak the language. What about the period here?
```suggestion
> 圧縮率の高いファイルアーカイバ。
```

**Review by navarroaxel — APPROVED** [review]

File: pages.ja/common/7zr.md (hunk 2 above)
Comment (Ry0taK) [reply pages.ja/common/7zr.md]: Thank you for the review! After checking other Japanese translations, I noticed that I should use the period in the end of description. So, I've added it!

**Pull request merged**